File: P1/E3/BodaBuilder.java
package E3;

class BodaBuilder extends EventoBuilder {
    
    public void construirNombre(String nombre) {
        evento.setNombre(nombre);
    }

    public void construirFecha(String fecha) {
        evento.setFecha(fecha);
    }

    public void construirUbicacion(String ubicacion) {
        evento.setUbicacion(ubicacion);
    }
}

File: P3/lib/boda_builder.dart
import 'evento_builder.dart';

class BodaBuilder extends EventoBuilder {
  @override
  void construirNombre(String nombre) {
    evento.nombre = nombre;
  }

  @override
  void construirFecha(String fecha) {
    evento.fecha = fecha;
  }

  @override
  void construirUbicacion(String ubicacion) {
    evento.ubicacion = ubicacion;
  }
}
